Packages/doc/daephys.rst

```diff
@@ -0,0 +1,33 @@
+.. _daephys:
+
+Da_Ephys
+========
+
+GUI control of data acquisition is achieved using the DA_Ephys panel. For
+example, the user can assign a stimulus set to a digital to analog (DA)
+channel. The DA_Epys panel centralizes control of multiple hardware components
+necessary for performing electrophysiology experiments, including the DAC,
+amplifier, and pressure regulators.
```

Comment:
The DA_Epys panel centralizes control data acquisition, including multiple hardware components necessary for performing electrophysiology experiments, such as the DAC, amplifier, and pressure regulators.